prow/config.yaml

```diff
@@ -121,9 +121,9 @@ presubmits:
             path: /mnt/disks/ssd0
 
   - name: pull-kubernetes-cross
-    context: pull-kubernetes-cross
-    rerun_command: "@k8s-bot pull-kubernetes-cross test this"
-    trigger: "@k8s-bot pull-kubernetes-cross test this"
+    context: Jenkins cross build
+    rerun_command: "@k8s-bot cross test this"
```

Comment:
I think it was "test" because we may have wanted to make it a required status at one point?